packages/back-end/src/validators/feature-revision-log.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from "zod";
+import { baseSchema } from "back-end/src/models/BaseModel";
+import { eventUser } from "./events";
+
+export const featureRevisionLog = z
+  .object({
+    featureId: z.string(),
+    version: z.number(),
+    user: eventUser,
+    approvedBy: eventUser.optional(),
+    timestamp: z.date(),
```

Comment:
There is already a `dateCreated` field in base model, so no need for `timestamp` here.